auth/jwt/http.go

```diff
@@ -12,23 +13,27 @@ import (
 func (f *Factory) HTTPAuthHandler(proxy auth.Proxy) http.Handler {
 	return http.HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
 		credential, err := io.ReadAll(request.Body)
-		if err != nil { //nolint:staticcheck,revive
-			// ...
+		if err != nil {
+			http.Error(writer, fmt.Errorf("failed to read request body credential: %w", err).Error(), http.StatusBadRequest)
```

Comment:
```suggestion
			http.Error(writer, fmt.Errorf("failed to read request body credential: %w", err).Error(), http.StatusInternalServerError)
```

I'd rather consider that as a 500 as the error doesn't depend on the request's content

Reply:
Okay, I considered the possibility that an error in reading the content of the body could be a client error with an invalid file, but it could also be a server error that is unable to read it.